samples/src/continuation/continuation.html

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<html 
+    xmlns="http://www.w3.org/1999/xhtml" 
+    xmlns:iso4217="http://www.xbrl.org/2003/iso4217" xmlns:xbrli="http://www.xbrl.org/2003/instance" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:link="http://www.xbrl.org/2003/linkbase" xmlns:ix="http://www.xbrl.org/2013/inlineXBRL" xmlns:ixt="http://www.xbrl.org/inlineXBRL/transformation/2011-07-31" xmlns:xbrldi="http://xbrl.org/2006/xbrldi" xmlns:ifrs="http://xbrl.ifrs.org/taxonomy/2017-03-09/ifrs-full"
+>
+    <head>
+        <title>iXBRL Viewer Continuation Document</title>
+        <style type="text/css">
+        </style>
+    </head>
+    <body xmlns="http://www.w3.org/1999/xhtml">
+        <div style="display: none">
+            <ix:header>
+    <ix:references>
+        
+        <link:schemaRef xlink:type="simple" xlink:href="http://xbrl.ifrs.org/taxonomy/2017-03-09/full_ifrs_entry_point_2017-03-09-es.xsd" />
+        
+        <link:schemaRef xlink:type="simple" xlink:href="http://xbrl.ifrs.org/taxonomy/2017-03-09/full_ifrs_doc_entry_point_2017-03-09.xsd" />
+        
+    </ix:references>
+    <ix:resources>
+      
+      
+        <xbrli:context id="c1">
+          <xbrli:entity>
+              <xbrli:identifier scheme="http://www.example.com/1234">1234</xbrli:identifier>
+              
+          </xbrli:entity>
+          <xbrli:period> 
+            
+                <xbrli:startDate>2018-01-01</xbrli:startDate>
+                <xbrli:endDate>2018-12-31</xbrli:endDate>
+            
+          </xbrli:period>
+          
+
+        </xbrli:context>
+      
+        <xbrli:context id="c2">
+          <xbrli:entity>
+              <xbrli:identifier scheme="http://www.example.com/1234">1234</xbrli:identifier>
+              
+          </xbrli:entity>
+          <xbrli:period> 
+            
+                <xbrli:startDate>2017-01-01</xbrli:startDate>
+                <xbrli:endDate>2017-12-31</xbrli:endDate>
+            
+          </xbrli:period>
+          
+
+        </xbrli:context>
+      
+    </ix:resources>
+</ix:header>
+        </div>
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Do we need all these extra lines?

Reply:
The files are generated using a templating system that's a bit too generous with its whitespace.  I've run it through a pretty printer in 8f96941 .  At some point I'll get the templater wired into the sample builder, but it's not quite ready yet.